system/handlers/renderers/content/Date.cfc

```diff
@@ -2,9 +2,17 @@ component output=false {
 
 	public string function default( event, rc, prc, args={} ){
 		var data = args.data ?: "";
+		var adminUserDetails = event.getAdminUserDetails();
+		var dateFormatMask   = translateResource( uri="cms:dateFormat");
+		
+		if ( IsStruct(adminUserDetails) ) {
+			if ( StructKeyExists(adminUserDetails, "user_admin_date_format") && Len(adminUserDetails.user_admin_date_format) ) {
+				dateFormatMask = adminUserDetails.user_admin_date_format;
+			}
+		}
```

Comment:
Can we just replace both if statements with:

`if ( Len( adminUserDetails.user_admin_date_format ?: "" ) ) {}`